src/timedata.cpp

```diff
@@ -105,11 +105,11 @@ void AddTimeData(const CNetAddr& ip, int64_t nOffsetSample)
                 }
             }
         }
-        if (fDebug) {
-            BOOST_FOREACH(int64_t n, vSorted)
-                LogPrintf("%+d  ", n);
-            LogPrintf("|  ");
-        }
-        LogPrintf("nTimeOffset = %+d  (%+d minutes)\n", nTimeOffset, nTimeOffset/60);
+        
+        BOOST_FOREACH(int64_t n, vSorted)
+            LogPrint("net", "%+d  ", n);
+        LogPrint("net", "|  ");
```

Comment:
Yes, it'll need fixing if that change is made in the future. 
Does anybody run in the non-specific `-debug` mode that would require this?

    2015-11-23 08:06:14 Added time data, samples 5, offset -1 (+0 minutes)
    2015-11-23 08:06:14 -1  -1  -1  +0  +0  |  nTimeOffset = -1  (+0 minutes)

Maybe that second print can go away entirely? The utility for debugging seems low.